### PR TITLE
Feature/employer jobs page

### PR DESCRIPTION
**Pull request by dmytro-chushko**
has combined redux jobs.api (getJobsByEmployer has added), modified employer-jobs-container, pagination, job-card
image

### CODE REVIEW

File: libs/components/src/lib/pagination/pagination.tsx [hunk 1]
```diff
@@ -1,27 +1,26 @@
-import styled from "styled-components";
-import React, { useEffect, useState } from "react";
-import ReactPaginate, { ReactPaginateProps } from "react-paginate";
-import { Container, StyledReactPagination } from "./pagination.styled";
-import { Items, JobsInterface } from "./items";
+import { useState } from "react";
+import { StyledReactPagination } from "./pagination.styled";
+import { JobsInterface } from "redux/jobs";
+import { Items } from "./items";
```
Comment (KseniaKoms) [inline libs/components/src/lib/pagination/pagination.tsx:4]: imports

Comment (dmytro-chushko) [reply libs/components/src/lib/pagination/pagination.tsx:4]: fixed

File: libs/components/src/lib/job-card/job-card.tsx [hunk 2]
```diff
@@ -1,72 +1,83 @@
...
-						<strong>{workExperience}</strong>
-						<strong>{levelEnglish}</strong>
-					</StyledJobCardParagraph>
+				<strong>{createdDate}</strong>
+				{userType === "freelancer" && (
+					<StyledButton buttonColor="redGradient" buttonSize="lg" fontSize="md">
+						<strong>{t("jobCard.sendProposal")}</strong>
+					</StyledButton>
+				)}
+				{userType === "employer" && (
```
Comment (lyudad) [inline libs/components/src/lib/job-card/job-card.tsx:50]: Снимок экрана 2022-12-19 в 18 54 28

**Comment by lyudad** [comment]
Снимок экрана 2022-12-19 в 18 56 00

**Pull request merged**